README.md

```diff
@@ -569,3 +571,61 @@ When you create a release branch in the form of `release/yyyy-mm-dd` or `hotfix/
 The CI pipeline will check to make sure you have done this to prevent any mistakes - if you have made a release or hotfix branch it will check that the value in the RELEASE file matches or not
 
 This is because it will use that value to tag the commit once its been merged into main as a reference point, and this is how it tracks which release it is as github actions struggles with post merge branch identification
+
+## Bastion
+
+For Dev and Test there is a bastion in place that can be used to connect to the RDS instance either through the CLI or from your chosen database client for Postgres.
+
+There are a few steps to get this sorted:
+
+1. Create a .pem file and adding it to your ssh agent:
```

Comment:
Only comment is if you make the key pair in AWS that it auto downloads for you which my smooth brain didnt realise haha - Also can you download a keypair once its been created? if not how would you get the persistent dev one if its already been made?

Reply:
I have created [NRLF-558](https://nhsd-jira.digital.nhs.uk/browse/NRLF-558) which will improve the process of storing a pem file for non-prod environments.

Reply:
I've also left instructions in the README that you can just set the `bastion_enabled` to `false` which will void the creation of the bastion for local dev.